### PR TITLE
Adds footer

### PR DESCRIPTION
**Pull request by ashpil**
This is a simple commit that just adds a footer.

It might look pretty weird until we have some content, though:

image

### CODE REVIEW

File: app/views/pages/rules.html.haml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.text
+  = "Placeholder"
```
Comment (ashpil) [inline app/views/pages/rules.html.haml:2]: Don't believe we have anything to put here just yet? Are we going to do the whole `config/locales` for large chunks of text like this, too?

File: config/locales/navigation.en.yml [hunk 2]
```diff
@@ -5,3 +5,8 @@ en:
       submit: "Submit"
       your_submissions: "Your submissions"
       inbox: "Inbox"
+  footer:
+      links:
+          about: "About"
+          rules: "Rules"
+          source: "GitHub"
```
Comment (ashpil) [inline config/locales/navigation.en.yml:12]: Not sure what else we need atm

File: spec/requests/pages_request_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (ashpil) [inline spec/requests/pages_request_spec.rb:1]: The command I used to add a controller just generated this spec page, which I assume is fine? Do we need any tests for the footer?

Comment (ashpil) [reply spec/requests/pages_request_spec.rb]: Nvm CircleCI didn't like this so I just chucked it

**Review by hernanat — APPROVED** [review]
LGTM after rebase (if needed) + squash

**Review by hernanat — APPROVED** [review]

**Pull request merged**